javascript/src/examples/register-address.ts

```diff
@@ -6,7 +6,7 @@ import { KeyringPair } from '@polkadot/keyring/types';
 import { GenericEventData } from '@polkadot/types/';
```

Comment:
I think this entire directory should be deleted. @jfrank-summit can you confirm?

Reply:
Let's do a Cleanup Javascript PR instead of doing it in this one.